inc/emergency-banner.php

```diff
@@ -2,61 +2,87 @@
 
 if( function_exists('get_field') ) {
```

Comment:
Please could we move this file to partials directory with other page elements

Reply:
Done